src/sqlancer/ReducerExample.java

```diff
@@ -0,0 +1,89 @@
+package sqlancer;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.BiFunction;
+import java.util.stream.Collectors;
+
+import sqlancer.FoundBugException.Reproducer;
+import sqlancer.common.query.Query;
+import sqlancer.sqlite3.SQLite3GlobalState;
+
+public class ReducerExample<G extends GlobalState<O, ?, C>, O extends DBMSSpecificOptions<?>, C extends SQLancerDBConnection>
```

Comment:
Perhaps `SimpleReducer` or `StatementReducer` is a better name? No need to address this in this PR.

Reply:
Done.